code/game/objects/items/stacks/stack_recipe.dm

```diff
@@ -24,27 +24,27 @@
 	src.on_floor = on_floor
 	src.window_checks = window_checks
 
-/datum/stack_recipe/proc/post_build(var/obj/item/stack/S, var/obj/result)
+/datum/stack_recipe/proc/post_build(var/obj/item/stack/S, obj/result)
```

Comment:
Can you remove the other `var/X`s from this file as well? Atleast for the procs you touched